AccountingSystem/Service/DataStoreHandler.py

```diff
@@ -146,7 +150,10 @@ def export_compactDB( self ):
     """
     Compact the db by grouping buckets
     """
-    return self.__acDB.compactBuckets() #pylint: disable=no-member
+    if self.runBucketing:
+      return self.__acDB.compactBuckets() #pylint: disable=no-member
+    else:
+      return RPCClient('Accounting/DataStoreMaster').compactDB()
```

Comment:
Please put a note to refer to documentation here (this is not necessarily a fixed name)